src/System.Net.WebSockets/ref/System.Net.WebSockets.csproj

```diff
@@ -15,4 +15,11 @@
     <ProjectReference Include="..\..\System.Security.Principal\ref\System.Security.Principal.csproj" />
     <ProjectReference Include="..\..\System.Threading.Tasks\ref\System.Threading.Tasks.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard' OR '$(TargetsNetFx)' == 'true'">
+    <Reference Include="mscorlib" />
+    <Reference Include="netstandard" />
+    <Reference Include="System.Memory" />
+    <Reference Include="System.Threading.Tasks.Extensions" />
+    <ProjectReference Include="..\..\Microsoft.Bcl.AsyncInterfaces\ref\Microsoft.Bcl.AsyncInterfaces.csproj" />
+  </ItemGroup>
```

Comment:
This shouldn't be necessary:
https://github.com/dotnet/corefx/blob/74a13ffba6d662decb073f2851a98435b80a65a2/src/System.Net.WebSockets/ref/Configurations.props#L4-L5